web/src/js/components/Modes/CaptureSetup.tsx

```diff
@@ -60,7 +60,7 @@ function ServerDescription({
     } else {
         desc = (
             <>
-                {full_spec.includes("local") ? (
+                {full_spec.startsWith("local") ? (
```

Comment:
Are you telling me we have `type` available but you are using `full_spec.startsWith`?  😛

Reply:
you're right 😅